### DISCUSSION TITLE
[Solved] Checkboxes missing ?

### DISCUSSION THREAD

**Discussion by mgkai** [JPTGamesAndApps/ShresthaFiles#2] [Q&A]
Hello, I activated checkboxes setting, thinking they may show up in the area that I marked yellow -- wrong thinking? TNX!

![image](https://user-images.githubusercontent.com/6969769/107439893-13863a00-6b33-11eb-89f5-6541979ed2f6.png)

**Comment by JPTGamesAndApps** [comment]
Hello mgkai,

In the settings pane you showed, you should see the checkbox to the left of the checkbox icon (just like the ones in "Use Generic Folder Thumbnail". Please click the Show Checkboxes again to see if the checkbox appears. Let us know if that solves your problem!

**Reply by mgkai** [reply]
Oh my... got it! I somehow took the checkbox icon / graphic as the actual checkbox...
